composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~8.1.10 || ~8.2",
+        "php": "~8.1.10 || ~8.2 || ~8.3",
```

Comment:
`~8.1.10 || ~8.2.0 || ~8.3.0` - otherwise composer will let install this even on 8.4 due to how it is handling `~`